docs/router/gRPC/plugins/library-features/logging.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: 'Logging'
+description: 'Enable and configure logging in gRPC plugins'
+icon: 'file-lines'
+---
+
+## Overview
+
+The Cosmo Router plugin framework provides built-in logging capabilities that integrate seamlessly with your plugin implementation. You can enable structured logging with different output formats and log levels to help with debugging and monitoring your plugins.
```

Comment:
But only for Go.

Reply:
We already mention in the plugin docs that we are limited to using Go as a plugin language. But I can put it there

Reply:
Got it. Do you think we find a better structure to reflect this in the docs better? Separating specific Go capabilities from Plugins concepts?

Reply:
Technically we have the structure already. Plugins are a separate subitem in left navigation bar and the library is also a subitem. We could eventually change it to be plugin-go-sdk and plugin-java-sdk etc. I think for now as we only support Go it makes no sense yet.